.gitignore

```
@@ -1,3 +1,4 @@
 result*
 /.direnv
 target
+/.pre-commit-config.yaml
```

```suggestion
.pre-commit-config.yaml
```
I would change it, so that even when you work in a subdirectory (e.g. rust/lanzatool), the .pre-commit file is ignored. I've had this issue before.

---

Do you have a specific scenario? Does pre-commit place configs in other directories?

---

It has happened in the past, but I can't reproduce it right now. So it should be fine.